salt/redash/files/env

```diff
@@ -0,0 +1,14 @@
+PYTHONUNBUFFERED=0
+REDASH_LOG_LEVEL=INFO
+REDASH_REDIS_URL=redis://redis:6379/0
+REDASH_COOKIE_SECRET={{ salt['pillar.get']('redash:cookie_secret') }}
+REDASH_SECRET_KEY={{ salt['pillar.get']('redash:secret_key') }}
+REDASH_DATABASE_URL=postgresql://{{ salt['pillar.get']('redash:postgres:username') }}:{{ salt['pillar.get']('redash:postgres:password') }}@{{ salt['pillar.get']('redash:postgres:hostname') }}/{{ salt['pillar.get']('redash:postgres:database') }}
```

Comment:
The current server also sets `POSTGRES_PASSWORD` but not sure if it's actually used by Redash.

Reply:
This variable is not used AFAICS.